app/presenters/templates/components/form.latte

```diff
@@ -4,7 +4,7 @@
 </ul>
 
 <table>
-<tr n:foreach="$form->controls as $input" n:class="$input->required ? required">
+<tr n:foreach="$form->controls as $input" n:if="$input->getOption(type) !== hidden" n:class="$input->required ? required">
 	<th>{label $input /}</th>
 	<td>{input $input} <span class=error n:ifcontent>{$input->error}</span></td>
```

Comment:
While true, it's not something we can prevent, unless we want to add complexity.

Also, this is a really nice showcase of `n:ifcontent` macro, which would stay hidden otherwise.